src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -843,5 +871,18 @@ public class ClassWithNotSupportedDictionaryButIgnored
         {
             [JsonIgnore] public Dictionary<int, int> MyDictionary { get; set; }
         }
+
+        public class Regression38565_Parent
+        {
+            public string Test { get; set; }
+            public Dictionary<string, string> Dict { get; set; }
```

Comment:
We should add some more test cases with different types like a collection (`List<T>` maybe), and also a user-defined POCO that also happened to be null (unless we have that already).

Reply:
I believe we already have explicit ones in the null test suite.